corehq/apps/locations/views.py

```diff
@@ -122,6 +123,7 @@ class LocationsListView(BaseLocationView):
 
     @use_jquery_ui
     @location_safe
+    @method_decorator(require_can_edit_commcare_users)
```

Comment:
@sheelio and I offlined, and it sounds like this should go in `locations_access_required`, as no locations views should be accessible to those users.